docs/components/modeler/web-modeler/run-or-publish-your-process.md

```diff
@@ -45,6 +45,11 @@ To deploy, click **Deploy** in the upper right corner of the modeling screen:
   Consider using a [process application](process-applications.md) that allows you to deploy a process and all dependent files together in a single bundle.
   :::
 - Implement and run your [job workers](../../concepts/job-workers.md) if you use tasks such as service or send tasks.
+- Ensure that there are no missing secrets, or misconfigured clients that are required for the process to run.
+  :::info
+  When missing secrets or misconfigured client credentials are detected, you will see a warning in the **Deploy diagram**.
```

Comment:
How do we know it's misconfigured? AFAIK its missing (unless you check for the Zeebe scope) - which could be useful to mention.
"When missing secrets or no client credentials with ______ scope are detected" would be more informative